src/At.vue

```diff
@@ -46,10 +46,6 @@ export default {
       type: Boolean,
       default: true
     },
-    members: {
-      type: Array,
-      default: () => []
-    },
```

Comment:
@piscitelli91 I'm confused why the prop of `members` is deleted 😕 it will break the members passing in